doc/_includes/channel_types.rst

```diff
@@ -65,4 +65,8 @@ ias            Internal Active Shielding data
 
 syst           System status channel information
                (Triux systems only)
+
+temperature    Temperature                               Celsius
```

Comment:
```suggestion
temperature    Temperature                               Degrees Celsius
```

Reply:
Because `temp` is ambiguous to me. More often than not (even to a non-programmer) I think when they see it they will think "temporary" moreso than "temperature".